src/mcp_server_helm/commands/release.py

```
@@ -0,0 +1,320 @@
+
+logger = logging.getLogger(__name__)
+
+
+def helm_install(chart: str, release_name: Optional[str] = None, namespace: Optional[str] = None,
+                 values_file: Optional[str] = None, set_values: Optional[Dict[str, str]] = None,
+                 description: Optional[str] = None, timeout: Optional[str] = None,
+                 wait: bool = False, atomic: bool = False) -> str:
+    """
+    Installs a Helm chart.
+    """
+    logger.info(f"Running helm install with chart={chart}, release_name={release_name}, namespace={namespace}")
```

Pylint (3.3.7)

[convention] Line too long (111/100)

(C0301)